app/assets/javascripts/backbone/views/loan_charts_view.coffee

```diff
@@ -111,7 +111,7 @@ class MS.Views.LoanChartsView extends Backbone.View
     data.addRow ["Fixed Costs", @breakevenData["total_fixed_costs"]]
 
     options = @defaultChartOptions
-    chart = new google.visualization.PieChart(document.getElementById('breakeven-costs-chart'))
+    chart = new google.visualization.PieChart(document.getElementById("breakeven-costs-chart-#{@id}"))
```

Comment:
Can you refactor to use `@$(...)` instead of getElementById. Can you also please refactor all these very similar lines into one helper method.